lib/scarpe/wv/star.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+class Scarpe
+  class WebviewStar < Scarpe::WebviewWidget
+    def initialize(properties)
+      super(properties)
+    end
+
+    def element(&block)
+      HTML.render do |h|
+        h.div(id: html_id, style: style) do
+          h.svg(width: @outer, height: @outer, style: "fill:#{fill_color};") do
+            h.polygon(points: star_points, style: "stroke:#{stroke_color};stroke-width:2")
+          end
+          block.call(h) if block_given?
+        end
+      end
+    end
+
+    private
+
+    def style
+      {
+        width: Dimensions.length(@width),
+        height: Dimensions.length(@height),
+        # "background-color": "red",
```

Comment:
Do we want to remove the background color comment?

Reply:
oh yeah will remove